docs/examples/alpine/templates/alpine-pod.yaml

```diff
@@ -3,8 +3,8 @@ kind: Pod
 metadata:
   name: {{.Release.Name}}-{{.Chart.Name}}
   labels:
-    heritage: {{.Release.Service}}
-    chartName: {{.Chart.Name}}
+    heritage: {{.Release.Service | quote }}
```

Comment:
Thanks @technosophos:

* `"{{.foo}}"` is accepted
* `{{.Chart.Name}}-{{.Chart.Version}}` is actually ignored. I will extend the linter to tell the user to use surrounding quotes